core/templates/dev/head/domain/utilities/UrlInterpolationServiceSpec.ts

```diff
@@ -304,15 +304,9 @@ describe('URL Interpolation Service', function() {
   });
 
   it('should interpolate URLs not requiring parameters', function() {
-    expect(uis.getStoryUrl('/storyId', {})).toBe('/story/storyId');
-    expect(uis.getStoryUrl('/storyId123', {})).toBe('/story/storyId123');
-    expect(uis.getStoryUrl('/story&Id', {})).toBe('/story/story&Id');
-    expect(function() {
-      uis.getStoryUrl('', {});
-    }).toThrowError('Empty path passed in method.');
-    expect(function() {
-      uis.getStoryUrl('storyId', {});
-    }).toThrowError('Path must start with \'/\': \'storyId\'.');
+    expect(uis.getStoryUrl('storyId')).toBe('/story/storyId');
+    expect(uis.getStoryUrl('storyId123')).toBe('/story/storyId123');
+    expect(uis.getStoryUrl('story&Id')).toBe('/story/story&Id');
```

Comment:
This seeems wrong, right? We should maybe double check the incoming story ID for valid chars and throw an error if it's invalid.

Reply:
Done, deleted this function and used a template.